### PR TITLE
DOCS 428: document the frame system editor

### PR DESCRIPTION
**Pull request by sguequierre**
* document accessing the frame system editor
* document example configuration
* add intro image to page from eng. 
* document display options and bounding geometries
* change name of configuration sub-pages to make the two examples a bit easier to understand while trying not to remove all references to static/dynamic
* remove quotes

### CODE REVIEW

File: docs/services/frame-system/_index.md [hunk 1]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+You can visualize how your robot is oriented in the Frame System in [the Viam app](https://app.viam.com).
+Navigate to the **config** tab on your robot's page, select **mode** as **Builder**, and click on **Frame System**.
+
+A 3D visualization of the spatial configuration of the reference frames of components configured on your robot is shown:
+
+![Default frame system configuration grid visualization for a single component, shown in the Frame System Editor](img/frame_system_basic.png)
+
+This tab provides a simple interface for simultaneously viewing and editing the position, orientation, and geometries of a robot's components in the Frame System.
+
+For example:
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:110]: Unsure why all of this is in bullet points - it's all one example

File: docs/services/frame-system/_index.md [hunk 2]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+
+- You have configured a robot with a [`jetson` board](/components/board), wired to a [`webcam` camera](/components/camera/webcam) and a [`wheeled` base](/components/base/wheeled) with two [motors](/components/motor) driving its wheels.
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:112]:
```suggestion
- Consider a robots configured with a [`jetson` board](/components/board), wired to a [`webcam` camera](/components/camera/webcam) and a [`wheeled` base](/components/base/wheeled) with two [motors](/components/motor) driving its wheels.
```

File: docs/services/frame-system/_index.md [hunk 3]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+
+- You go to the **config** tab of your robot and click on the **Frame System** sub-tab.
+You have not yet added any reference frame configuration to these components, so they are all shown to be located on the default `world` origin point as follows:
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:115]: here I'd suggest reframing this as "If you go to ..."

File: docs/services/frame-system/_index.md [hunk 4]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+
+  ![Camera oriented .5 degrees OX shown in the Frame System Editor](img/demo_camera_edited_2.png)
+
+Now that the Frame System is configured so that the spatial visualization for your robot matches its spatial orientation in real life, you can utilize [Motion Service](/services/motion) methods that take in reference frame information.
+
+### Display Options
+
+Click and drag on the **Frame System** visualization to view the display from different angles, and pinch to zoom in and out:
+
+{{<gif webm_src="img/frame_system_demo.webm" mp4_src="img/frame_system_demo.mp4" alt="The frame system visualization zooming in and out around the example robot with a camera, board, and wheeled base.">}}
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:140]: this is very cool - for the future, open up the developer tools and remove the chat bubble when taking these screen recordings (let me know if you want me to show you how). Not worth redoing now, just keep it in mind

File: docs/services/frame-system/_index.md [hunk 5]
```diff
@@ -11,6 +11,8 @@ tags: ["frame system", "services"]
 
 Any robot configured in Viam comes with the Frame System service: an internally managed and mostly static system for storing the "reference frame" of each component of a robot within a coordinate system configured by the user.
 
+![Visualization of a wheeled base configured with motors and a mounted camera in the Frame System tab of the Viam app UI](img/frame_system_wheeled_base.png)
+
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:15]: On line 21 - can you add where you click add frame? on the component panel I think?

File: docs/services/frame-system/_index.md [hunk 6]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
-- [A component attached to another, dynamic, component](/services/frame-system/component-on-dynamic)
+- [A Reference Frame:](/services/frame-system/frame-config) A component attached to a static surface
+- [Nested Reference Frames:](/services/frame-system/nested-frame-config) A component attached to another, dynamic, component
+
+### Visualize the Frame System
+
+You can visualize how your robot is oriented in the Frame System in [the Viam app](https://app.viam.com).
+Navigate to the **config** tab on your robot's page, select **mode** as **Builder**, and click on **Frame System**.
+
+A 3D visualization of the spatial configuration of the reference frames of components configured on your robot is shown:
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:104]: The Viam app shows you...

File: docs/services/frame-system/_index.md [hunk 7]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+{{% tab name="Orthographic Camera" %}}
+
+![Non-default Orthographic Camera view shown in the Frame System Editor](img/demo_orthographic.png)
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Bounding Geometries
+
+Add `Geometry` properties to the **Frame System** [configuration](#configuration) of a component to visualize bounding geometries by selecting a component and selecting a **Geometry** type.
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:159]: Can you specify where I do that?

File: docs/services/frame-system/_index.md [hunk 8]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+
+By default, a **Geometry** is shown surrounding the origin point of a component as follows:
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:161]: can you describe this? Maybe "as a cube around the component"

**Review by npentrel — COMMENTED** [review]
This is so cool - love all the screenshots. Just minor feedback really

File: docs/services/frame-system/_index.md (hunk 4 above)
Comment (sguequierre) [reply docs/services/frame-system/_index.md:140]: Oh I didn't think of that thank you!

File: docs/services/frame-system/_index.md (hunk 5 above)
Comment (sguequierre) [reply docs/services/frame-system/_index.md:15]: Yep!

File: docs/services/frame-system/_index.md (hunk 8 above)
Comment (sguequierre) [reply docs/services/frame-system/_index.md:161]: it's not necessarily a box though!

File: docs/services/frame-system/_index.md [hunk 9]
```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+
+{{< tabs name="Visualize Adding Geometry Bounds" >}}
+{{% tab name="Box" %}}
+
+![Demo robot with default box bounds added to the wheeled base, shown in the Frame System Editor](img/demo_bound_box.png)
+
+You can adjust the **Size** and **Translation** of a **Geometry** to change these bounds.
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/services/frame-system/_index.md:168]: It doesn't sound too odd to me but happy to change it, easiest way I can think of to refer to the feature

File: docs/services/frame-system/_index.md (hunk 7 above)
Comment (sguequierre) [reply docs/services/frame-system/_index.md:159]: done, it does look... chunky though! 😆

File: docs/services/frame-system/_index.md (hunk 8 above)
Comment (sguequierre) [reply docs/services/frame-system/_index.md:161]: Not necessarily default as box but removed as follows

File: docs/services/frame-system/_index.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/services/frame-system/_index.md:169]: It is a little weird though

File: docs/services/frame-system/_index.md (hunk 6 above)
Comment (sguequierre) [reply docs/services/frame-system/_index.md]: Done

File: docs/services/frame-system/_index.md [hunk 10]
```diff
@@ -91,18 +94,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+
+  ![Example robot's default frame configuration shown in the Frame System Editor. All components are stuck on top of each other](img/demo_base_unedited.png)
+
+You measure the distance on the floor from your base to your board and camera setup and find it to be 200 millimeters.
+If you add this value to `"X"` in your base's reference frame `Translation` attribute, the Frame System readjusts to show the base's translation:
+
+  ![Base translated 200mm forwards shown in the Frame System Editor](img/demo_base_edited.png)
+
+Next, you measure the distance from your board to the camera mounted overhead and find it to be 50 millimeters.
+If you add this value to `"Z"` in your camera's reference frame `Translation` attribute, the Frame System readjusts to show the camera's translation:
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:126]: Since this is all an example I find the `you` strange in all of this.

**Review by npentrel — APPROVED** [review]
I'd change the remaining `yous` in the example and make it more describing what the example is instead of roleplaying that the user has that in front of them. Otherwise LGTM

File: docs/services/frame-system/_index.md (hunk 10 above)
Comment (sguequierre) [reply docs/services/frame-system/_index.md:126]: Yeah it's a little weird.

 I don't know how to translate directions like `You have not yet added any reference frame configuration to these components.` without speaking to the user or using *we*

File: docs/services/frame-system/_index.md [hunk 11]
```diff
@@ -91,18 +94,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
...
+## How the Frame System Works
 
 `viam-server` builds a tree of reference frames for your robot with the `world` as the root node and regenerates this tree following reconfiguration.
 
 Access a [topologically-sorted list](https://en.wikipedia.org/wiki/Topological_sorting) of the generated reference frames in the robot's logs at `--debug` level:
 
 ![an example of a logged frame system](img/frame_sys_log_example.png)
 
-Consider the example of a [component attached to a dynamic component](/services/frame-system/component-on-dynamic): a robotic arm, `A`, attached to a gantry, `G`, which in turn is fixed in place at a point on the `World` of a table.
+Consider the example of nested reference frame configuration where a [two dynamic components are attached](/services/frame-system/nested-frame-config): a robotic arm, `A`, attaches to a gantry, `G`, which in turn is fixed in place at a point on the `World` of a table.
```
Comment (sguequierre) [inline docs/services/frame-system/_index.md:194]:
```suggestion
Consider the example of nested reference frame configuration where [two dynamic components are attached](/services/frame-system/nested-frame-config): a robotic arm, `A`, attaches to a gantry, `G`, which in turn is fixed in place at a point on the `World` of a table.
```

**Pull request merged**